conans/test/integration/remote/download_retries_test.py

```diff
@@ -58,4 +58,4 @@ def get(self, *args, **kwargs):
                             requester_class=BuggyRequester)
         client.run("install --reference=pkg/0.1@lasote/stable", assert_error=True)
         self.assertEqual(str(client.out).count("Waiting 0 seconds to retry..."), 2)
-        self.assertEqual(str(client.out).count("Error 200 downloading"), 3)
+        self.assertEqual(str(client.out).count("Error 200 downloading"), 4)
```

Comment:
This increments by 1 because the detailed information about the Graph error includes the error downloading message.